thamos/lib.py

```diff
@@ -1296,7 +1302,7 @@ def get_verified_packages_from_static_analysis(
                 exc.status,
                 exc.body,
             )
-            continue
+            raise ApiError(error_message)
```

Comment:
You can register a special exception for this (Thamos specific exception): https://github.com/thoth-station/thamos/blob/master/thamos/exceptions.py